BSPConvert.Lib/Source/BSPConverter.cs

```diff
@@ -990,21 +999,26 @@ private int CreatePatchFace(Vertex[] faceVerts, int faceIndex)
 			return sourceBsp.Faces.Count - 1;
 		}
 
-		private void CreatePatchDisplacement(int faceIndex, Vertex[] faceVerts, int patchWidth, int patchStartVertex)
+		private void CreatePatchDisplacement(int sFaceIndex, Vertex[] faceVerts, int patchWidth, int patchStartVertex, Face qFace)
 		{
 			var data = new byte[Displacement.GetStructLength(sourceBsp.MapType)];
 			var displacement = new Displacement(data, sourceBsp.Displacements);
 
 			var power = options.DisplacementPower;
+			var texture = qFace.Texture.Name;
```

Comment:
You can clean up this code a bit by extracting lines 1008 - 1012 into a method called `GetMinTesselation()` and set `displacement.MinimumTesselation = GetMinTesselation()`